commandline/src/main/java/tech/pegasys/web3signer/commandline/config/PicoV3WalletBulkloadParameters.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2022 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.commandline.config;
+
+import tech.pegasys.web3signer.commandline.DefaultCommandValues;
+import tech.pegasys.web3signer.signing.config.KeystoresParameters;
+
+import java.nio.file.Path;
+
+import picocli.CommandLine.Option;
+
+public class PicoV3WalletBulkloadParameters implements KeystoresParameters {
```

Comment:
I think if we want to regard wallets as a type of keystore then the CLI arg should reflect that and also be called --keystores-path. That would mean it's the mode that determines the reading of the keystore, rather than the option name.

Reply:
done. Modified various instance variables/class names to use `v3Keystores` as well (instead of `wallet`).